src/yb/util/wait_state.cc

```diff
@@ -22,6 +22,27 @@ namespace yb::util {
 // __thread WaitStateInfoPtr WaitStateInfo::threadlocal_wait_state_;
 thread_local WaitStateInfoPtr WaitStateInfo::threadlocal_wait_state_;
 
+void AUHMetadata::QueryIdHexToUInt(std::string queryId) {
```

Comment:
Is this supposed to return the UInt query_id?

Reply:
sounds like Update/Set QueryIdFromHex may be a better name.

also use query_id_str instead of CamelCase

Reply:
Yeah, this will update the uint query_id in AUHMetadata. Updated the PR with your comments